dashboard/db/migrate/20151022174100_create_puzzle_ratings.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePuzzleRatings < ActiveRecord::Migration
+  def change
+    create_table :puzzle_ratings do |t|
+      t.references :user, index: true
+      t.integer :script_id
```

Comment:
Yeah, I have no particular preference for references, just wondering.

On Tue, Oct 27, 2015 at 3:08 PM, Elijah Hamovitz <notifications@github.com>
wrote:

> In dashboard/db/migrate/20151022174100_create_puzzle_ratings.rb
> <https://github.com/code-dot-org/code-dot-org/pull/4753#discussion_r43194863>
> :
>
> > @@ -0,0 +1,15 @@
> > +class CreatePuzzleRatings < ActiveRecord::Migration
> > +  def change
> > +    create_table :puzzle_ratings do |t|
> > +      t.references :user, index: true
> > +      t.integer :script_id
>
> They're functionally identical, right? I just find it misleading that
> references by default create neither indexes nor foreign keys.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/4753/files#r43194863>.
>